lib/graphql/fragments.js

```diff
@@ -0,0 +1,80 @@
+import { gqlV1 } from '.';
+
+export const invoiceFields = gqlV1`
```

Comment:
Is this understood by `eslint-plugin-graphql` ?

Reply:
I think so, in the frontend it works when we use the `gqlV2` helper so it should work with this one. But `eslint-plugin-graphql` is not yet plugged on this repo, so I can't tell for sure.

Reply:
Just double checked and it effectively works with `eslint-plugin-graphql` but not with `prettier`, would be interesting to dig.

Reply:
Related: https://github.com/prettier/prettier/pull/4395

Reply:
Seems a workaround is writing: ```gqlV2/* GraphQL */ ```